fio.h

```diff
@@ -460,7 +460,7 @@ struct thread_data {
 	CUdevice  cu_dev;
 	CUcontext cu_ctx;
 	CUdeviceptr dev_mem_ptr;
-#endif	
```

Comment:
Also unrelated - it's clearly correct, but it should at least be a separate commit, not bundled with this.